fedimint-cli/src/ng.rs

```diff
@@ -103,6 +99,38 @@ pub async fn handle_ng_command<D: IDatabase>(
 
             Ok(serde_json::to_value(notes).unwrap())
         }
+        ClientNg::LnInvoice {
+            amount,
+            description,
+            expiry_time,
+        } => {
+            let mut dbtx = client.db().begin_transaction().await;
+            let active_gateway = client
+                .fetch_active_gateway(&mut dbtx.with_module_prefix(LEGACY_HARDCODED_INSTANCE_ID_LN))
+                .await?;
+            dbtx.commit_tx().await;
+
+            let operation_id = client
+                .create_bolt11_invoice_and_receive(amount, description, expiry_time, active_gateway)
```

Comment:
Would be nice if this function also returned the invoice, too. Then callers could postentially return here and present the invoice, and listen for updates later.

Reply:
Easy enough change. It is also available in the operation log (so listeners to state machine updates can get it from there)

Reply:
ahh ok. maybe i just don't understand the API yet.

Reply:
It's fine to return it here too. I made the change.